build.gradle

```diff
@@ -148,33 +149,60 @@ tasks.register('checkAkitInstall', JavaExec) {
     classpath = sourceSets.main.runtimeClasspath
 }
 
-tasks.register('installLocalGitHookPostCheckout', Copy) {
-    from new File(rootProject.rootDir, '.scripts/post-checkout')
-    into { new File(rootProject.rootDir, '.git/hooks') }
-    filePermissions {
-        unix(0775)
-    }
-    doLast {
-        println("---Added Post Checkout Git Hook Script---")
+def ensureSymlink(link, target) {
+    if (!Files.isSymbolicLink(link)) {
+        if (Files.exists(link)) {
+            Files.delete(link)
+        }
+
+        Files.createSymbolicLink(link, target)
+
+        def relative = { f -> rootDir.toPath().relativize(f) }
+        println("--- Installed Symbolic Link ${relative(link)} -> ${relative(target)} ---")
     }
 }
 
-tasks.register('installLocalGitHookPreCommit', Copy) {
-    from new File(rootProject.rootDir, '.scripts/pre-commit')
-    into { new File(rootProject.rootDir, '.git/hooks') }
-    filePermissions {
-        unix(0775)
-    }
+// Setup local git settings
+def gitConfig = 'Git Configuration'
+tasks.register('installLocalGitHookSymlinks') {
+    group = gitConfig
+
+    outputs.dir "$rootDir/.git/hooks/"
+
     doLast {
-        println("---Added Pre Commit Git Hook Script---")
+        def linkDir = file("$rootDir/.git/hooks/").toPath()
+        def gitHooks = fileTree("$rootDir/.scripts/git-hooks/")
+        gitHooks.forEach { File target ->
+            def link = linkDir.resolve(target.toPath().getFileName())
```

Comment:
maybe save:
hooksDir = "$rootDir/.git/hooks/"
scriptsDir = "$rootDir/.scripts/git-hooks/"

and use the variables in the function